pkg/worker/logger.go

```diff
@@ -103,10 +103,12 @@ func (r *ContainerLogger) CaptureLogs(containerId string, outputChan chan common
 				instance.LogBuffer.Write([]byte(msg.Message))
 			}
 
-			if msg.TaskID != nil && msg.Message != "" {
-				log.Printf("<%s>:<%s> - %s\n", containerId, *msg.TaskID, msg.Message)
-			} else if msg.Message != "" {
-				log.Printf("<%s> - %s\n", containerId, msg.Message)
+			for _, line := range strings.Split(msg.Message, "\n") {
```

Comment:
This doesn't affect user code but it affects the logs captured by fluent-bit from workers for `snoop`. Some lines won't have container-ids since if they had a newline so we must split the multiline logs and add them ourselves. Shouldn't increase # of ES entries there are